include/exiv2/config.h

```diff
@@ -62,7 +62,7 @@
 
 ///// Path separator macros      /////
 #ifndef EXV_SEPARATOR_STR
-#if defined(WIN32) && !defined(__CYGWIN__)
+#if defined(_WIN32)
```

Comment:
This one might actually be needed? Not 100% sure, but I don't think Cygwin/MSYS understand Windows paths, but they do also define _WIN32 as far as I remember? (potentially applies to all `!defined(__CYGWIN)` occurences below).

Reply:
Cygwin does not define _WIN32

Reply:
Indeed - bad memory here, should've checked first...